src/components/masters/program/ProgramPage.jsx

```diff
@@ -95,11 +95,12 @@ class ProgramPage extends Component {
                     <Hero
                       programTitle={programName}
                       organizationLogo={{
-                        url: 'https://www.edx.org/sites/default/files/school/image/logo/gtx-logo-200x101.png',
-                        alt: 'Georgia Tech Institute of Technology logo',
+                        url: `${process.env.DESIGNER_BASE_URL}${programBranding.organization_logo.url}`,
```

Comment:
`organization_logo.url` is going to be absolute URL (at least in prod). 
Stage response:
```
"organization_logo": {
    "url": "https://stage-edx-designer.s3.amazonaws.com:443/original_images/gt_logo.png",
    "alt": ""
},
```